server/setup/traefik-setup.ts

```diff
@@ -184,6 +184,13 @@ export const createDefaultTraefikConfig = () => {
 	const yamlStr = dump(configObject);
 	mkdirSync(MAIN_TRAEFIK_PATH, { recursive: true });
 	writeFileSync(mainConfig, yamlStr, "utf8");
+	
+	const acmeJsonPath = "/etc/dokploy/traefik/dynamic/acme.json";
+	if (existsSync(acmeJsonPath)) {
+	    chmodSync(acmeJsonPath, '600');
+	} else {
+	    console.error(`File not found: ${acmeJsonPath}`);
+	}
 };
```

Comment:
Looks good, I tested locally and it works, can you move this code above the validation if the mainConfig exists? because for the current instances of dokploy running that validation will never reach to run